wix-bookings/guides/Wix Bookings Life Cycle.md

```diff
@@ -0,0 +1,33 @@
+# Wix Bookings Lifecycle
+
+To manage bookings, a life cycle status is used to track their progression through different stages. This status is essential for controlling the possible actions that can be taken, such as confirming a pending booking. By utilizing this life cycle status, the system effectively regulates the booking process from start to finish.
+
+A booking can go through checkout for payment or have no payment, in any case, the validation occurs through [Confirm or decline](https://dev.wix.com/api/rest/wix-bookings/confirmation/confirm-or-decline-booking), where availability, payment and the need for business confirmation are checked.
+
+A booking does not need payment to be confirmed, but payment will make sure that the booking is confirmed, even in a case of `DOUBLEBOOKING`, another way of allowing `DOUBLEBOOKING` is passing `skipAvailability` as `true` when creating the booking. 
+
+The following table describes all the possible status for a booking:
+
+| Status | Definition |
+|---------------|-------------------------|
+
+| `CREATED` | The booking has been created, but doesn't yet appear in the business calendar. 
+The first step and status of a booking is created, it can be used for checkout or processed without it.
```

Comment:
This is not a sentence - it should be two sentences or rewritten to be one sentence.  I think if you ran every sentence through ChatGPT you'd be in a better place.
I do not understand what this means: "it can be used for checkout or processed without it." 
Do we always say "business calendar?" Can we just always say "calendar?"
I don't understand what "the first step and status" means, can be worded better.